### DISCUSSION TITLE
ERROR when restoring

### DISCUSSION THREAD

**Discussion by BabaTataKaka** [etcd-io/etcd#18952] [Q&A]
Hi there,

I'm trying to test the ectd restore command but for some reason I'm getting an error:

```bash
root@server1:/tmp/etcd-2024-11-26_00:00:01# etcdutl snapshot restore snapshot.db --data-dir /var/lib/etcd_2

2024-11-26T15:20:21Z	info	snapshot/v3_snapshot.go:260	restoring snapshot	{"path": "snapshot.db", "wal-dir": "/var/lib/etcd_2/member/wal", "data-dir": "/var/lib/etcd_2", "snap-dir": "/var/lib/etcd_2/member/snap"}

Error: seek snapshot.db: invalid argument
````

Not sure from where the "invalid argument" error is coming from. The snapshot.db file is in the current folder where I'm running the command.

The ectd cluster was reset from scratch and the /etcd-2024-11-26_00:00:0 backup folder was copied on the new leader node where the restore is performed.

**Comment by jmhbnz** [comment]
Hi @BabaTataKaka thanks for raising this.  I haven't seen this error before during a restore, can you please confirm:
- Operating system
- CPU architecture
- etcd version

Just to double check are you certain that the `snapshot.db` file is present and is a valid snapshot?

**Reply by BabaTataKaka** [reply]
I tried to restore from the original/untouched folder and it worked. Look like the restore command does like to restore from a folder that was copied.

Operating system Ubuntu 22.04
CPU architecture x64
etcd version 3.5.13
